content/02.main-text.md

```diff
@@ -1,3 +1,21 @@
 ## Main text {.page_break_before}
 
-*Work in progress*
+### Introduction
+
+Open research – which includes sharing code, data, and manuscripts – benefits the researchers who engage in open practices [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+`TODO: more references needed`
+Here we advocate for the benefits of writing review articles openly, specifically planning, organizing, writing, and editing this type of manuscript collaboratively in an unrestricted public forum.
```

Comment:
At this point, do we want to simply report on our experience? In the conclusion perhaps we should advocate.

`Here, we report on the process of writing a review article openly: the planning, organizing, writing, and editing occurred collaboratively in a public forum where participants were free to join as they wished.`